spec/controllers/admin/tracks_controller_spec.rb

```diff
@@ -383,12 +376,16 @@
       end
     end
 
+    before :each do
+      conference.venue = create(:venue)
+      @room = create(:room, venue: conference.venue)
+    end
+
     context 'has start_date, end_date and room' do
       before :each do
-        self_organized_track.start_date = Date.today
-        self_organized_track.end_date = Date.today
-        conference.venue = create(:venue)
-        self_organized_track.room = create(:room, venue: conference.venue)
+        self_organized_track.start_date = Date.current
```

Comment:
For this case you can simply define a variable, using let, named full_self_organized_track or something similar, that indicates that this is a self_organized track with all of its attributes.